index.js

```diff
@@ -114,10 +121,10 @@ class Plugin extends PluginBtp {
   async _handleMoney (from, { requestId, data }) {
     const primary = data.protocolData[0]
     if (primary.protocolName === 'machinomy') {
-      const payment = new Payment(JSON.parse(primary.data.toString()))
+      const payment = JSON.parse(primary.data.toString())
```

Comment:
We should probably wrap this type of parsing in a try/catch in case the other side sends something invalid

Reply:
Looks like `ilp-plugin-btp` should catch any parsing errors: https://github.com/interledgerjs/ilp-plugin-btp/blob/master/index.js#L273 -- and that's what triggers sending a BTP error packet. Do we want something here as well?

Reply:
Nah, that's fine then